Description: Bump the internal version, missed in the new upstream release
Author: Simon Quigley <tsimonq2@ubuntu.com>
Forwarded: https://github.com/palinek/nm-tray/issues/29
Last-Update: 2018-07-05
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -3,7 +3,7 @@ project(nm-tray)
 cmake_minimum_required(VERSION 3.1.0 FATAL_ERROR)
 cmake_policy(SET CMP0071 NEW)
 
-set(NM_TRAY_VERSION "0.3.0")
+set(NM_TRAY_VERSION "0.4.0")
 
 set(QT_MIN_VERSION "5.4.0")
 set(KF5_MIN_VERSION "5.36.0")
